webpack.requirejs_jquery_q.config.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
```

Comment:
General: A good test (if you haven't done it already) would be to drop this in the sign-in-widget and see if everything works properly.